card/mod/01_core/set/all/content.rb

```diff
@@ -35,6 +34,10 @@ def clean_html?
   true
 end
 
+def history?
+  false
+end
```

Comment:
What does this mean?  (Seems like it means that we never store histories, but that can't be right, can it?)